src/lib.rs

```diff
@@ -645,6 +672,15 @@ mod tests {
         verify_single_value_matches(res, string_value);
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_get_registered_service_names() {
+        let consul = get_client();
+        let _res = consul
```

Comment:
can we register a service & validate that it's actually there?

Reply:
sure I can add it as part of this pr since it increases the robustness of the test itself.